pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -386,6 +386,49 @@ type ObjectStoreSpec struct {
 
 	// The rgw pod info
 	Gateway GatewaySpec `json:"gateway"`
+
+	// The multisite info
+	Multisite MultisiteSpec `json:"multisite"`
+}
+
+type GatewaySpec struct {
```

Comment:
nit: can you move this struct lower so it's easier to see the diff? I didn't think the gateway properties would change much.